cmd/edgectl/notify.go

```diff
@@ -7,10 +7,21 @@ import (
 	"github.com/datawire/ambassador/pkg/supervisor"
 )
 
-var notifyRAI *RunAsInfo
+var (
+	notifyRAI     *RunAsInfo
+	notifyEnabled = false
```

Comment:
Did I miss a way to turn this back on?

Reply:
Oh, good point. Not everything in my tree got committed; I should have left this out too. That said, notifications don't work on Linux (any longer), so I'm not too fussed about leaving it off until the next RC. Is that okay?

Reply:
I'm OK with leaving it for the next RC. Let's talk about it later; I think the notifications could be very useful if tweaked a bit.